README.md

```diff
@@ -188,13 +199,21 @@ When retrieving an image URL (via `getMapUrl()`) with effects applied, an error
       redRange: {from: 0.2, to: 0.8},
       greenRange: {from: 0.2, to: 0.8},
       blueRange: {from: 0.2, to: 0.8},
+      customEffect: {
+        redFunction: pixelValue => exampleFunction[pixelValue],
+        greenFunction: pixelValue => exampleFunction[pixelValue],
+        blueFunction: pixelValue => exampleFunction[pixelValue],
+      }
```

Comment:
I think the interface would be more intuitive (and powerful) if there was only a single function available which could modify all three color bands. Sth. like this:
```
customEffect: ({r, g, b}) => exampleFunction(r, g, b),
```

If you decide to do so, an interesting (and short) `exampleFunction` could be one that rotates the colors:
```
const rotateColors = ({r, g, b}) => ({g, b, r});
```

Reply:
Thanks, I changed it to your suggestion. Don't really know why I went with separated functions the first time

Reply:
Looks good, except documentation still needs to be updated in lines 202-206 (and I would suggest using a simpler `exampleFunction`).

Reply:
Changed the documentation to the simplest possible function.